src/System.Management.Automation/engine/parser/ast.cs

```diff
@@ -5472,50 +5472,88 @@ public class PipelineAst : PipelineBaseAst
         }
 
         /// <summary>
+        /// Construct a pipeline from a collection of commands.
+        /// </summary>
+        /// <param name="extent">The extent of the pipeline.</param>
+        /// <param name="pipelineElements">The collection of commands representing the pipeline.</param>
+        /// <exception cref="PSArgumentNullException">
+        /// If <paramref name="extent"/> is null.
+        /// </exception>
+        /// <exception cref="PSArgumentException">
+        /// If <paramref name="pipelineElements"/> is null or is an empty collection.
+        /// </exception>
+        public PipelineAst(IScriptExtent extent, IEnumerable<CommandBaseAst> pipelineElements) :this (extent, pipelineElements, backgroundProcess: false)
+        {
+
+        }
+
+        /// <summary>
         /// Construct a pipeline from a single command.
         /// </summary>
         /// <param name="extent">The extent of the pipeline (which should be the extent of the command).</param>
         /// <param name="commandAst">The command for the pipeline.</param>
+        /// <param name="backgroundProcess">Indicates that this pipeline should be run in the background</param>
         /// <exception cref="PSArgumentNullException">
         /// If <paramref name="extent"/> or <paramref name="commandAst"/> is null.
         /// </exception>
-        public PipelineAst(IScriptExtent extent, CommandBaseAst commandAst)
+        public PipelineAst(IScriptExtent extent, CommandBaseAst commandAst, bool backgroundProcess)
             : base(extent)
         {
             if (commandAst == null)
             {
                 throw PSTraceSource.NewArgumentNullException("commandAst");
             }
 
+            this.BackgroundProcess = backgroundProcess;
             this.PipelineElements = new ReadOnlyCollection<CommandBaseAst>(new CommandBaseAst[] { commandAst });
             SetParent(commandAst);
         }
 
         /// <summary>
+        /// Construct a pipeline from a single command.
+        /// </summary>
+        /// <param name="extent">The extent of the pipeline (which should be the extent of the command).</param>
+        /// <param name="commandAst">The command for the pipeline.</param>
+        /// <exception cref="PSArgumentNullException">
+        /// If <paramref name="extent"/> or <paramref name="commandAst"/> is null.
+        /// </exception>
+        public PipelineAst(IScriptExtent extent, CommandBaseAst commandAst) :this (extent, commandAst, backgroundProcess: false)
+        {
+
+        }
+
+        /// <summary>
         /// A non-null, non-empty collection of commands that represent the pipeline.
         /// </summary>
         public ReadOnlyCollection<CommandBaseAst> PipelineElements { get; private set; }
 
         /// <summary>
+        /// Indicates that this pipeline should be run in the background.
+        /// </summary>
+        public bool BackgroundProcess { get; private set; }
```

Comment:
>BackgroundProcess [](start = 20, length = 17)

Maybe the property should not include `Process`, e.g. if in the future, we support in-process jobs, then it's just in the background, not necessarily in another process.